data_transfer/subset_matrix_table.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python3
+
+"""
+Takes a path to a MatrixTable, and optionally sample IDs
+Reads in the MT from provided input path
+If sample IDs are specified, subset containing only those
+Write the [reduced] MT into Test
+Optionally with the --vcf flag, write as a VCF
+Optionally with both --chr and --pos specified, subset to a specific locus
+"""
+
+from argparse import ArgumentParser
+import logging
+import sys
+
+import hail as hl
+
+from cpg_utils.hail_batch import output_path, init_batch
+from cpg_utils.config import get_config
+
+
+def subset_to_samples(
+    matrix: hl.MatrixTable, samples: list[str], remove_hom_ref: bool
```

Comment:
As a general style note, most Hail code I've seen just uses `mt` to denote a matrix table and typically reuses the same variable, e.g. `mt = mt.filter_cols(hl.set(samples).contains(mt.s))`.